node/ci/unit_tests.sh

```diff
@@ -5,5 +5,5 @@ CI_DIR="$( cd "$( dirname "$0" )" && pwd )"
 export RUST_BACKTRACE=full
 export RUSTFLAGS="-D warnings -Anon-snake-case"
 pushd "$CI_DIR/.."
-cargo test --release --lib --no-fail-fast -- --nocapture --skip _integration
+cargo test --release --lib --no-fail-fast --features masq_lib/log_recipient_test -- --nocapture --skip _integration
```

Comment:
Not here but try to integrate --features masq_lib/no_test_share to integration_tests.sh